app/build.gradle

```diff
@@ -1,20 +1,7 @@
-buildscript {
-    repositories {
-        maven { url 'https://maven.fabric.io/public' }
-        jcenter()
-    }
-
-    dependencies {
-        classpath 'io.fabric.tools:gradle:1.28.0'
-        classpath 'com.github.triplet.gradle:play-publisher:1.2.0'
```

Comment:
Are we not depending on play publisher? What is taking its place?
I assume crashlytics is taking over fabric correct?